cmake_scripts/default_build_setup.cmake

```diff
@@ -14,23 +14,23 @@ endif()
 message(STATUS "Build type: ${CMAKE_BUILD_TYPE}")
 
 # The language standard we use
-add_definitions("-std=c++11")
+add_definitions("-std=c++17")
 
 # build shared instead of static libraries
 set(BUILD_SHARED_LIBS TRUE)
 
 option(BUILD_COVERAGE
        "Determines whether C++ files will be built with support for coverage measurement"
        OFF)
-option(DISABLE_CXX11_ABI
-       "Determines whether the C++ compiler shall link to a C++11 conforming standard library, see https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html for details"
+option(DISABLE_CXX17_ABI
+       "Determines whether the C++ compiler shall link to a C++17 conforming standard library, see https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html for details"
```

Comment:
We should not change this. It's only for C++11.

Reply:
I've reverted that